src/basic/virt.c

```diff
@@ -230,13 +230,26 @@ static int detect_vm_smbios(void) {
 static int detect_vm_dmi(void) {
 #if defined(__i386__) || defined(__x86_64__) || defined(__arm__) || defined(__aarch64__)
 
-        int r;
+        int r, r1;
```

Comment:
Style nits: we use k in these cases. Also all these new variables can be declared in the switch case block.

Reply:
Ok.